apps/myaccount/src/utils/common-utils.ts

```diff
@@ -73,5 +73,14 @@ export class CommonUtils {
         set(newPref, CommonConstants.SEEN_ANNOUNCEMENTS_KEY, [ ...seen, id ]);
 
         AppUtils.setUserPreferences(newPref);
-    }
+    }  
 }
+
+/**
+ * Converts the isReadOnlyUser string to a boolean variable
+ *
+ * @return {boolean} True/False
+ */
+export const convertToBoolean = (isReadOnlyUserString: string): boolean => {
```

Comment:
Shall we move this to profile utils as rename as `isProfileReadOnly`?

```suggestion
export const isProfileReadOnly = (state string): boolean => {
    return (state === "true");
}
```

Reply:
Fixed by abb00182f514f442fa084df945978362e7b37d65.